pyroscope.go

```diff
@@ -1,4 +0,0 @@
-package pyroscope
```

Comment:
- Fixed logging
- Done the workaround to allow both single and double dash flags as discussed
- fixed so that errors are not printed twice

> Why did you delete this file? It's needed for pkger to work.

As for this, it is needed to be a `.go` file in the root directory. I just replaced that one with another one so pkger should still be working (it worked when I ran it with `pkger`)